base/docs/basedocs.jl

```diff
@@ -2568,15 +2721,28 @@ julia> 2 * 7 * 8
 
 julia> *(2, 7, 8)
 112
+
+julia> [2 0; 0 3] * [1, 10]  # matrix * vector
+2-element Vector{Int64}:
+  2
+ 30
+
+julia> 1/2pi, 1/2*pi  # juxtaposition has higher precedence
+(0.15915494309189535, 1.5707963267948966)
+
```

Comment:
This example is already listed in the text above.

Reply:
Well, the other way around. Maybe the text is too long?